src/main/java/org/jenkinsci/test/acceptance/utils/SystemEnvironmentVariables.java

```diff
@@ -0,0 +1,28 @@
+package org.jenkinsci.test.acceptance.utils;
+
+/**
+ * Gets a system environment variable. First it checks if the variable is an environment variable and if it is the case,
+ * it returns the configured value.
+ * If not then it checks if it is defined as java system property, and if not it returns the default value.
+ */
+public class SystemEnvironmentVariables {
+
+    public static final String getPropertyVariableOrEnvironment(String property, String def) {
```

Comment:
FTR, I do not think there is any consistency at all. Envvars was inherited from the ruby times so some contributors kept using those, while a lot of new features started using properties.

Assuming the pom bestiality is needed just to support the old envvar approach, I see no reason not to move to properties (with some BC mechanism for existing envvars). I do not think this PR should address that but it would be nice if it used the mechanism we agree to use in the long run.